Dockerfile

```diff
@@ -21,11 +21,16 @@ ARG GOOS=linux
 # ARG GOARCH=amd64
 
 RUN go build -o /adam/bin/adam main.go
+RUN go get -u github.com/go-swagger/go-swagger/cmd/swagger
+RUN GO111MODULE=off swagger generate spec -o ./swaggerui/swagger.json
+
 
 FROM alpine:3.11
 
 COPY scripts /bin
 COPY samples /adam
+ADD swaggerui ./swaggerui/
+COPY swaggerui /bin/
```

Comment:
Why are we putting the same `swaggerui` dir in 2 separate locations in the container?

Reply:
Fixed